apps/condo/domains/banking/tasks/generateReports.js

```diff
@@ -253,17 +268,17 @@ const generateReports = async (taskId) => {
         }
         const lastReport = reports[reports.length - 1]
         if (reports.length > 0) {
-            if (!isEqual(
-                lastReport.data, data) ||
-                Math.round(Number(payload.totalIncome) * 10000) / 10000 !== Math.round(Number(lastReport.totalIncome) * 10000) / 10000 ||
-                Math.round(Number(payload.totalOutcome) * 10000) / 10000 !== Math.round(Number(lastReport.totalOutcome) * 10000) / 10000
-            ) {
+            const isEqualReportsData = !isEqual(lastReport.data, data)
+                || Math.round(Number(payload.totalIncome) * 10000) / 10000 !== Math.round(Number(lastReport.totalIncome) * 10000) / 10000
+                || Math.round(Number(payload.totalOutcome) * 10000) / 10000 !== Math.round(Number(lastReport.totalOutcome) * 10000) / 10000
```

Comment:
Best to explain this code in comments, like: we need to check that ... for that we use ...